### ISSUE TITLE
Cannot Find Name For Some Files

### ISSUE THREAD

**Issue by maestroh** [Microsoft/vscode#12145]
- VSCode Version: 1.5.2
- OS Version: Windows 10 Pro 1607
- Typescript Version: 1.8.10

Steps to Reproduce:

1. Open a Typescript project
2. Only the first file I navigate to can find modules

I have a Node project running with Typescript support. When I start Code, Typescript modules are only found for the first file I open. The remaining files have the "Cannot Find Name" error for all modules. In some cases the modules are the same ones on the first file, so I know the definitions are available. If I restart VS Code or run the reload window command, then the error goes away from the first file I open, and will now show the error for all others.

Any ideas on a fix?

**Comment by waderyan** [comment]
@maestroh thank you for opening this issue. Could you provide me with some code that I can copy and paste or clone to reproduce this behavior? 

I'm not understanding the context of "Cannot Find Name." If you could include a screenshot or a short video that would be helpful. 

Thank you.

**Comment by maestroh** [comment]
@waderyan Here you go. To add a bit of explanation to what you're seeing here: I've got two files open, middleware.ts and index.ts. middleware.ts is the first one I navigate to and the types are recognized. In index.ts nothing is recognized. Notice that both files import 'express'. I reload the window and the types are recognized in index.ts, but not middleware.ts. It seems that the types are only recognized in the first file I open or focus on. I may be doing something silly, but I can't think of what could be wrong here. I have tried restarting both VS Code and my machine.

![vscode-error](https://cloud.githubusercontent.com/assets/2009595/18605326/e98d584c-7c5c-11e6-8344-2c3f14683881.gif)
